index.html

```diff
@@ -1233,24 +1233,24 @@
 ## Data Minimization {#data-minimization}
 
 <div class="practice">
-<span class="practicelab">Sites, user agents, and other actors should minimize the amount of
-<a>personal data</a> they transfer between actors on the Web.</span>
+<span class="practicelab">[=Sites=], [=user agents=], and other [=actors=]
+should minimize the amount of [=personal data=] they transfer. [=User agents=]
+should share data only when needed, in alignment with a user's goals and
+interests. Web APIs should be designed to minimize the amount of data that
+sites need to request.</span>
 </div>
 
-Data minimization limits the risks of data being disclosed or misused, and it also helps
-user agents more meaningfully explain the decisions their users need to make.
+Data minimization limits the risks of data being disclosed or misused. It also
+helps [=user agents=] more meaningfully explain the decisions their users need
```

Comment:
Arguably not part of an editorial sweep but it helps any actor, not just UAs.

Reply:
That was the original text, but happy to accept a suggestion if you want to make it directly